CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # Changes
 
+* `SendError` now implements `ResponseError`.
```

Comment:
It is more suited for `Added` section rather than `Changes`.

Since we did add `ResponseError` impl for `actix::SendError`

Reply:
Ah, I missed that section. I'll fix it when I get a chance.

Reply:
I'll correct it myself